### ISSUE TITLE
Debug Tools Error

### ISSUE THREAD

**Issue by LepuavkoDV** [ElvUI-WotLK/ElvUI#170]
1x Blizzard_DebugTools-1.0\Blizzard_DebugTools.lua:432: bad argument #6 to 'format' (string expected, got nil)
Blizzard_DebugTools-1.0\Blizzard_DebugTools.lua:432: in function <...e\AddOns\Blizzard_DebugTools\Blizzard_DebugTools.lua:418>
<in C code>: in function `ScriptErrorsFrame_Update'
Blizzard_DebugTools-1.0\Blizzard_DebugTools.lua:384: in function <...e\AddOns\Blizzard_DebugTools\Blizzard_DebugTools.lua:383>
<in C code>: in function `Show'
Blizzard_DebugTools-1.0\Blizzard_DebugTools.lua:411: in function <...e\AddOns\Blizzard_DebugTools\Blizzard_DebugTools.lua:387>
<in C code>: in function `ScriptErrorsFrame_OnError'
ElvUI_AddOnSkins-1.01\core.lua:372: in function `CallSkin'
ElvUI_AddOnSkins-1.01\core.lua:408: in function <ElvUI_AddOnSkins\core.lua:396>
<in C code>: ?
ElvUI-5.19\core\core.lua:313: in function `InitializeModules'
ElvUI-5.19\core\core.lua:936: in function `?'
CallbackHandler-1.0-6:146: in function <...ack\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:146>
<string>:"safecall Dispatcher[1]":4: in function <[string "safecall Dispatcher[1]"]:4>
<in C code>: ?
<string>:"safecall Dispatcher[1]":13: in function `?'
CallbackHandler-1.0-6:91: in function `Fire'
AceEvent-3.0-3 (DrDamage):120: in function <...e\AddOns\DrDamage\libs\AceEvent-3.0\AceEvent-3.0.lua:119>

Locals:
editBox = ScriptErrorsFrameScrollFrameText {
 fontSize = 13
 0 = <userdata>
 cursorHeight = 0
 cursorOffset = 0
}
index = 1
_ScriptErrorsFrame = ScriptErrorsFrame {
 index = 1
 close = <unnamed> {}
 messages = <table> {}
 0 = <userdata>
 previous = <unnamed> {}
 lastButton = <unnamed> {}
 indexLabel = <unnamed> {}
 next = <unnamed> {}
 firstButton = <unnamed> {}
 order = <table> {}
 count = <table> {}
 times = <table> {}
 locals = <table> {}
 template = "Transparent"
 title = <unnamed> {}
 seen = <table> {}
}
ERROR_FORMAT = "Message: %s
Time: %s
Count: %s
Stack: %s
Locals: %s"

**Comment by LepuavkoDV** [comment]
this error appears only if you enable gearscore skin. otherwise itsok

**Comment by Bunny67** [comment]
what is your version gearscore

**Comment by LepuavkoDV** [comment]
3.1.17 (Release)

**Comment by Dissectional** [comment]
GearScore is a known buggy addon and shouldn't be used, if you disable everything but GearScore you'll probably find you still get errors.

GearScore Lite is a good alternative.  
[Here](http://www.truewow.org/forum/viewtopic.php?t=6355) is a thread with download links to some 3.3.5 addons including GearScore Lite.
I've also used [Elitist Group](http://www.wowinterface.com/downloads/info15573-ElitistGroup.html)  which is similar.
Elitist Group never received Cataclysm support so the last version works in 3.3.5.  
Finally, while looking for this information for you, I came across [WoWEquip](https://www.wowace.com/addons/wowequip/), which looks interesting and I'm going to give it a go now.
